### ISSUE TITLE
Switch concurrency warnings to complete

### ISSUE THREAD

**Issue by maxgoedjen** [maxgoedjen/secretive#523]
Once Xcode 15.4 is out.

**Comment by maxgoedjen** [comment]
@PBHDK yeah, that's it – any time you see Sendability mentioned that's related to this, not anything wrong with your toolchain.

**Comment by PBHDK** [comment]
@maxgoedjen oh, that's unfortunate. 

Is there an obvious workaround I'm missing? ATM, I'm not able to build because of this ...

**Comment by maxgoedjen** [comment]
@PBHDK for the moment, you can turn down the concurrency warnings – there's a few in Xcode settings, and a few in the package. Try setting them to "minimal" or "targeted".

**Comment by PBHDK** [comment]
Will do. Thanks, @maxgoedjen !
